src/traversal/__tests__/writeRelayUpdatePayload-test.js

```diff
@@ -735,7 +735,7 @@ describe('writePayload()', () => {
       edgeID = generateClientEdgeID(connectionID, commentID);
     });
 
-    it('warns if the created `edge` field is missing in the payload', () => {
+    it('warns if the created `edge` field is missing in the payload and was never queried by client', () => {
```

Comment:
nit: 80 char limit, also maybe abbreviate to

    it('warns if a new edge was never queried by client', () => {